packages/driver/cypress/e2e/commands/actions/click.cy.ts

```diff
@@ -282,7 +283,7 @@ describe('src/cy/commands/actions/click', () => {
       const button = cy.$$('#button')
 
       cy.get('#button').click().then(($button) => {
-        expect($button).to.match(button)
+        expect($button[0]).to.eq(button[0])
```

Comment:
`to.match` is only meant to compare via Regex, but we were using this in a lot of places to check equality of elements. It did seem to be passing anyways somehow.